src/app/shared/components/validation-hint/validation-hint.component.html

```diff
@@ -11,24 +11,35 @@
   </ng-container>
 </small>
 
-<ng-template #requiredInputTmpl> Це поле є обов'язковим. <br /> </ng-template>
+<ng-template #requiredInputTmpl> {{ 'FORMS.VALIDATIONS.REQUIRED_INPUT' | translate }} <br /> </ng-template>
 
-<ng-template #invalidCharactersTmpl> Перевірте введені дані, використовуйте будь-ласка тільки кирилицю <br /> </ng-template>
+<ng-template #invalidCharactersTmpl>
+  {{ 'FORMS.VALIDATIONS.INVALID_INPUT' | translate }}{{ 'FORMS.VALIDATIONS.INVALID_CHARACHTERS' | translate }}<br />
+</ng-template>
 
-<ng-template #invalidSymbolsTmpl> Перевірте введені дані, використовуйте будь-ласка тільки кирилицю та символи ( ' - ) <br /> </ng-template>
+<ng-template #invalidSymbolsTmpl>
+  {{ 'FORMS.VALIDATIONS.INVALID_INPUT' | translate }}{{ 'FORMS.VALIDATIONS.INVALID_SYMBOLS' | translate }} <br />
+</ng-template>
 
-<ng-template #invalidLengthTmpl> Це поле має містити від {{ minCharachters }} до {{ maxCharachters }} символів. <br /> </ng-template>
+<ng-template #invalidLengthTmpl>
+  {{ 'FORMS.VALIDATIONS.INVALID_LENGTH_START' | translate }} {{ 'FROM' | translate | lowercase }} {{ minCharachters }}
+  {{ 'TO' | translate | lowercase }} {{ maxCharachters }} {{ 'FORMS.VALIDATIONS.INVALID_LENGTH_END' | translate }} <br />
```

Comment:
![image](https://user-images.githubusercontent.com/27493959/205054369-210ab66c-e207-4773-9460-236713c27e51.png)
should be lower case